packages/cozy-client/src/models/dacc.spec.js

```diff
@@ -104,3 +104,37 @@ describe('dacc', () => {
     ).toThrow()
   })
 })
+
+describe('buildAggregateParams', () => {
+  beforeAll(() => {
+    jest
+      .spyOn(global.Date, 'now')
+      .mockImplementation(() => new Date(2022, 5, 1).valueOf())
+  })
+  it('should throw if measure name is not correct', () => {
+    expect(() => buildAggregateParams({})).toThrow()
+    expect(() => buildAggregateParams({ measurName: 42 })).toThrow()
+  })
+
+  it('should throw if dates are not correct', () => {
+    expect(() =>
+      buildAggregateParams({
+        measureName: 'test',
+        startDate: 'Lundi 10 août 2021'
+      })
+    ).toThrow()
+    expect(() =>
+      buildAggregateParams({
+        measureName: 'test',
+        endDate: 'Lundi 10 août 2021'
+      })
+    ).toThrow()
+  })
+  it('should build date if not provided', () => {
+    expect(buildAggregateParams({ measureName: 'test' })).toEqual({
+      measureName: 'test',
+      startDate: '1970-01-01T00:00:00.000Z',
+      endDate: new Date(2022, 5, 1).toISOString()
+    })
+  })
+})
```

Comment:
can you add a working test with at least a given startDate?

Reply:
Yes I can [914cd5d](https://github.com/cozy/cozy-client/pull/1186/commits/914cd5d5e5c82feb96a18e92234cb4ce685514b3)